### DISCUSSION TITLE
Allow changing POI and other object visibility on high zoom levels including searched ones

### DISCUSSION THREAD

**Discussion by ghost** [organicmaps/organicmaps#3294] [Ideas]
Currently I need to zoom in too much to see all the POIs and details in a particular area. I know this is probably done because of resource management and making the map less cluttered but when *searched* POIs get hidden, using the app does becomes a bit annoying unfortunately. So I suggest some kind of settings at the very least to solve this issue

Anyways I really like the project, it truly is filling a hole in the FOSS world. <3

**Comment by foersterp** [comment]
#2901 is about names of cities but could be related.

**Comment by biodranik** [comment]
There are several related issues already, please search for them. In short: we would love to do it, but it's not easy.
